FSW/src/headers/config.hpp

```diff
@@ -74,13 +85,40 @@ static TimedEvent sweepTimeoutEvent(SWEEP_TIMEOUT_MSEC);
 //      look at checkSweepChange() in timing.cpp for more info
 static Event sweepDirectionChangeEvent; 
 
+
 /* - - - - - - Command Handling Module - - - - - - */
 const int COMMAND_QUEUE_SIZE = 100;     // maximum number of commands the command queue can store.
 const int SERIAL_TIMEOUT_MSEC = 50;     // milliseconds, maximum time to wait for serial input
 
+
 /* - - - - - - Fault Mitigation Module - - - - - - */
-const int WD_RESET_INTERVAL = 100;  // watchdog feeding interval, ms
-const int WD_PULSE_DUR = 10;        // watchdog reset signal duration, MICROSECONDS!!!
+const int WD_RESET_INTERVAL_MSEC = 100;     // milliseconds, watchdog feeding interval
+const int WD_PULSE_DUR_MICROSEC = 10;       // microseconds, watchdog reset signal duration

```

Comment:
I like all of the standardization you did in `config.hpp`, makes it much clearer what the constants mean